server/controllers/banner.js

```diff
@@ -82,9 +86,9 @@ module.exports = {
       request({url: imageUrl, encoding: null}, (e, r, data) => {
         const contentType = r.headers['content-type'];
 
-        const imageHeight = 64;
+        const imageHeight = Math.round(maxHeight / 2);
```

Comment:
Because we are wrapping the image within a SVG of 64px height. And above we get the original image with a height of 128px (to make it retina).